jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -366,8 +368,18 @@ AuthConfiguration getTargetImageAuth() {
   boolean getUseCurrentTimestamp() {
```

Comment:
Let's add `@Deprecated` for tracking purposes. I tried to add `@Deprecated` as much as possible to the code related to `extraDirectory` that should be removed later, except for the code where you can't really miss removing them like tests that call a method that will certainly be removed or where the notion `@Deprecated` doesn't apply well.